### PR TITLE
Remove Items

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Removes "Add Tags" and "Print" menu options from the Project Menu. Also removes the "I Agree to Something" checkbox from Sign Up and ITSI Sign Up modals.

## Testing Instructions

Test the Project Menu while logged in, when logged out, and when viewing public projects as third-party.
Test the sign-up procedure.
Run all tests.

## Notes

Should I also remove the "Print" option from the Scenario Menu?

Connects #844 
Connects #838

### CODE REVIEW

**Comment by mmcfarland** [comment]
Looks good.  I poked around and even though we're sending an "agreed" value to the server, it looks like we're not persisting it anywhere.  I just want to make sure that we can distinguish in the future users who have signed up and not actually "agreed" after we do introduce a real ToS.

**Comment by mmcfarland** [comment]
> Should I also remove the "Print" option from the Scenario Menu

Yep, good catch.

**Comment by rajadain** [comment]
Re: distinguishing users who haven't explicitly agreed to a yet unwritten ToS, since we don't persist the agreement it will just have to be date of sign up vs the date we deploy the ToS.

Will remove "Print" from Scenario Menu as well.

**Comment by rajadain** [comment]
"Print" removed from Scenario Menu in https://github.com/rajadain/model-my-watershed/commit/e2dd613f69627443068b41a9ebb981832cf9f0a0

File: src/mmw/js/src/user/templates/baseModal.html [hunk 1]
```diff
@@ -16,6 +16,15 @@
     </div>
 {% endmacro %}
 
+{% macro hidden_checkbox(value, name, label='') %}
+    <div class="checkbox" style="display: none;">
+        <label>
+            <input name="{{ name }}" type="checkbox" id="{{ name }}" checked>
```
Comment (lliss) [inline src/mmw/js/src/user/templates/baseModal.html]: According to the card (emphasis added)
> Users signing up should **not** have this field marked as accepted

I think we probably will also want to clear out that dummy text as well.

**Comment by rajadain** [comment]
@lliss the ToS Agreement is **completely** removed in https://github.com/rajadain/model-my-watershed/commit/36c499eb5fe62fe75af504508407390f0e10b728. Is this closer to your interpretation of the card?

**Comment by lliss** [comment]
> @lliss the ToS Agreement is completely removed in rajadain@36c499e. Is this closer to your interpretation of the card?

It's fine with me since it's fairly easy to add back. I thought we'd leave the checkbox there but hidden and remove the silly "I agree to something" text and the validation checks for it. So with the exception of the deletion:

```
-{% macro hidden_checkbox(value, name, label='') %}
-    <div class="checkbox" style="display: none;">
-        <label>
-            <input name="{{ name }}" type="checkbox" id="{{ name }}" checked>
-            {{ label }}
-        </label>
-    </div>
-{% endmacro %}
``` 

this is pretty much what I had in mind.

**Comment by rajadain** [comment]
The [original checkbox macro](https://github.com/WikiWatershed/model-my-watershed/blob/4f16ea139a899eb85b9db81df2befd74e39220fc/src/mmw/js/src/user/templates/baseModal.html#L10-L17) is still there, so it should be easy to add it back.

This does look cleaner, and when we want to add a checkbox it may be easier to add it back to our then current style, rather than going back and un-commenting code, which is why I just outright deleted it. Git will preserve history.

**Comment by lliss** [comment]
I totally agree. +1

**Comment by lewfish** [comment]
+1, tested

**Comment by rajadain** [comment]
Just pushed https://github.com/rajadain/model-my-watershed/commit/470ea8a044355db08f000bf478d3a59056e22166 which addresses #862. Will squash and merge shortly.

**Pull request merged**